### PR TITLE
TopicRepository Added and ScienceId were removed

### PR DESCRIPTION
**Pull request by Muhammadjon05**

### CODE REVIEW

**Review by JamolMeyliyev — APPROVED** [review]

File: src/Sciences.API/Program.cs [hunk 1]
```diff
@@ -10,7 +12,7 @@
     .CreateLogger();
 
 builder.Logging.AddSerilog(logger);
-
+builder.Services.AddScoped<ITopicRepository, TopicRepository>();
```
Comment (sardorbekakhmedov) [inline src/Sciences.API/Program.cs:15]: Serviclarni ServiceCollectionExtensions fayliga qo'shgan yaxshiroq, chunki boshqa barcha servicelar o'sha yerda

File: src/Sciences.API/Repositories/ClassRepositories/TopicRepository.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+public class TopicRepository : ITopicRepository
+{
+    private readonly ScienceDbContext _context;
+
+    public TopicRepository(ScienceDbContext context)
+    {
+        _context = context;
+    }
+    public async Task<List<Topic>> GetTopics(Guid scienceId)
```
Comment (sardorbekakhmedov) [inline src/Sciences.API/Repositories/ClassRepositories/TopicRepository.cs:15]: Agar Method asinxron bo'lsa  mehtod nomini oxirida Async so'zi qo'yilish kerak, 
masalan GetTopicsAsync

**Comment by sardorbekakhmedov** [comment]
Code da conflicts bor main bilan solishtirib FIX qilish kerak

File: src/Sciences.API/Repositories/ClassRepositories/TopicRepository.cs (hunk 2 above)
Comment (Muhammadjon05) [reply src/Sciences.API/Repositories/ClassRepositories/TopicRepository.cs:15]: aha tushunarli

**Comment by javlonfattoev** [comment]
confilicni fix qilib merge qilavering

**Pull request closed without merging**